.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -113,6 +113,7 @@ This is a checklist for PR authors. Please make sure to complete all tasks and c
 - [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
 - [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
 - [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
+- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used.
```

Comment:
The added item in the reviewer checklist contains the following phrase: `- from a logged in and logged out account`

should we also add this phrase here because that's often the cause of the issue (from your OP)?